components/base-war/src/main/webapp/resources/uicomponents/widgets/userpicker/userPicker.js

```diff
@@ -128,7 +128,7 @@ widgets.UserPicker = Class.create(widgets.Suggest, {
       avatarWrapper.insert(new Element('span', {alt: data.info, 'class': data.icon}));
     }
     container.insert(avatarWrapper);
-    var userName = source.highlight ? this.emphasizeMatches(this.sInput, data.info) : data.info;
+    var userName = (source.highlight ? this.emphasizeMatches(this.sInput, data.info) : data.info).escapeHTML();
```

Comment:
`emphasizeMatches` probably adds some HTML tags to format the matching substring differently. Escaping should ve done before this formatting is applied.

Reply:
Yes, you're right. 
I also missed committing the spot after "This record is owned by"
Fixing...